src/signTxOutput.c

```diff
@@ -226,6 +226,8 @@ static void signTx_handleOutput_addressParams()
 	TRACE("Policy: %d", (int) policy);
 	ENSURE_NOT_DENIED(policy);
 
+	subctx->addressSecurityPolicy = policy;
```

Comment:
If I understand correctly, the `subctx` is initialized in `signTxOutput_init()` which sets all fields to zero, including the addressSecurityPolicy, which is an invalid value. Do you know how is it possible that Ledger did not reject the transaction?

Reply:
ah got it - the policies were written in a way that the zero value feel through to `SHOW()` :facepalm:

Reply:
or shown the tokens anyway, looking at the original logic of `policyForSignTxOutputAssetGroup`

Reply:
ah sorry, I realized the problem was indeed that the tokens defaulted to be shown, I thought the opposite was happening for some reason during the review and got confused